infra/deploy_kubernetes_manifests.tf

```diff
@@ -60,7 +60,7 @@ resource "kubernetes_job" "kubernetes_manifests_deployer_job" {
         service_account_name = local.k8s_service_account_name
         container {
           name  = "kubernetes-manifests-deployer"
-          image = "us-docker.pkg.dev/google-samples/containers/gke/kubernetes-manifests-deployer:v0.0.0.1"
+          image = "us-docker.pkg.dev/google-samples/containers/gke/kubernetes-manifests-deployer:v0.0.0.2"
```

Comment:
I have re-built the Docker image in the [`kubernetes_manifests_deployer`](https://github.com/GoogleCloudPlatform/terraform-ecommerce-microservices-on-gke/tree/main/kubernetes_manifests_deployer) folder, and pushed the image to [`us-docker.pkg.dev/google-samples/containers/gke/kubernetes-manifests-deployer:v0.0.0.2`](us-docker.pkg.dev/google-samples/containers/gke/kubernetes-manifests-deployer:v0.0.0.2).

Why is the version `0.0.0.2`?
Eventually, when versioning is set up for this GitHub repo as a whole, we will need to update the version of